pandas/tests/groupby/test_transform.py

```diff
@@ -1196,3 +1206,81 @@ def test_transform_lambda_indexing():
         ),
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_corrwith(transformation_func, df_for_transformation_func):
+
+    # GH 27905
+    df = df_for_transformation_func.copy()
```

Comment:
is a copy needed? By default, the dataframe fixture is reevaluated for each test unless scope is passed to fixture decorator. see https://docs.pytest.org/en/latest/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session

Reply:
thanks, yeh your right -  i had assumed default scope  was `session` - will update.